spec/services/rad_rate_limiter_spec.rb

```diff
@@ -22,7 +22,7 @@
       before { 4.times { rate_limiter.run { 'foo' } } }
 
       it 'raises error' do
-        expect { rate_limiter.run { 'foo' } }.not_to raise_error(described_class::RateLimitExceededError)
```

Comment:
totally unrelated, but fixed this annoying warning that showed up every test suite run